src/ring_theory/adjoin_root.lean

```diff
@@ -8,6 +8,7 @@ Adjoining roots of polynomials
 import data.polynomial.field_division
 import ring_theory.adjoin
 import ring_theory.principal_ideal_domain
+import linear_algebra.finite_dimensional
```

Comment:
Please make the changes in this file in a standalone PR

Reply:
#4830 which depends on #4829